frontend_app_workspace_advanced/src/container/WorkspaceAdvanced.jsx

```diff
@@ -478,7 +521,39 @@ export class WorkspaceAdvanced extends React.Component {
           default: this.sendGlobalFlashMessage(props.t('Error while adding the member to the space'), 'warning')
         }
         break
-      default: this.sendGlobalFlashMessage(props.t('Error while adding the member to the space', 'warning'))
+      default: this.sendGlobalFlashMessage(props.t('Error while adding the member to the space'), 'warning')
+    }
+  }
+
+  handleClickAcceptRequest = async userId => {
+    const { props, state } = this
+    const fetchPutSubscriptionAccept = await putSubscriptionAccept(
+      state.config.apiUrl,
+      state.content.workspace_id,
+      userId,
+      state.content.default_user_role
+    )
+    switch (fetchPutSubscriptionAccept.status) {
+      case 204: break
+      case 400:
+        switch (fetchPutSubscriptionAccept.body.code) {
+          case 3008: this.sendGlobalFlashMessage(props.t('This user already is in the space'), 'warning'); break
+          default: this.sendGlobalFlashMessage(props.t('Error while adding the member to the space'), 'warning')
+        }
+        break
+      default: this.sendGlobalFlashMessage(props.t('Error while adding the member to the space'), 'warning')
+    }
+  }
+
+  handleClickRejectRequest = async userId => {
+    const { props, state } = this
+    const fetchPutSubscriptionReject = await putSubscriptionReject(
```

Comment:
you need to call handleFetchResult on every async requests

Reply:
Updated